jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.vue

```diff
@@ -78,11 +78,11 @@
       action_label="Calculate"
       action_tooltip="Calculate moment map"
       :action_spinner="spinner"
-      :action_disabled="n_moment > 0 && output_unit_selected === 'Velocity' && reference_wavelength === 0"
+      :action_disabled="n_moment > 0 && output_unit_selected !== 'Spectral Unit' && reference_wavelength === 0"
       @click:action="calculate_moment"
     ></plugin-add-results>
 
-    <v-row v-if="n_moment > 0 && output_unit_selected === 'Velocity' && reference_wavelength === 0">
+    <v-row v-if="n_moment > 0 && output_unit_selected !== 'Spectral Unit' && reference_wavelength === 0">
       <span class="v-messages v-messages__message text--secondary" style="color: red !important">
           Cannot calculate moment: Must set reference wavelength for output in velocity units.
```

Comment:
I can get this same error message by setting n_moment to 1 and output_unit to Flux... granted that is only possible by the UI, so I'm not sure its worth an extra case here to show the correct error message or not.